cmd/titus-storage/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"log"
```

Comment:
Please use journalhook, consider using structured logging, aka zerolog?

Reply:
will do. This will be especially handy for tracing the lifetime of a particular volume id